html/captive-portal/templates/status/reset_password.html

```diff
@@ -0,0 +1,39 @@
+[% IF status == "success" %]
+<div class="media media--notice u-p u-mb">
+  <div class="media__img">[% flashIcon(level='notice') %]</div>
+  <p class="media__body">[% i18n("Your password has been updated") %]</p>
+</div>
+[% ELSIF status == "error_match" %]
+<div class="media media--error u-p u-mb">
+  <div class="media__img">[% flashIcon(level='error') %]</div>
+  <p class="media__body">[% i18n("The two entered passwords did not match") %]</p>
+</div>
+[% ELSIF status == "error_fill" %]
+<div class="media media--error u-p u-mb">
+  <div class="media__img">[% flashIcon(level='error') %]</div>
+  <p class="media__body">[% i18n("One of the password fields hasn't been filled") %]</p>
+</div>
+[% END %]
+
+[% UNLESS showLogin %]
```

Comment:
Instead of not showing the form in the template if the user isn't logged in, it should instead redirect to the login page if he isn't logged in and then here you unconditionally display the form